### PR TITLE
KubeCON24 recap

### PR DESCRIPTION
**Pull request by thomasdarimont**
A short recap of Keycloak usage at KubeCON / CloudNativeCon EU 2024 in Paris.

### CODE REVIEW

**Review by tnorimat — APPROVED** [review]
LGTM

**Comment by ahus1** [comment]
@stianst - I reviewed and copy-edited the post and it should now be good for publishing. I'll leave it here for another day before I'll merge this PR.

File: blog/2024/keycloak-at-kubecon-eu-2024-recap.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+:title: Recap from KubeCon + CloudNativeCon Europe 2024
+:date: 2024-04-12
```
Comment (stianst) [inline blog/2024/keycloak-at-kubecon-eu-2024-recap.adoc:2]:
```suggestion
:date: 2024-04-15
```

File: blog/2024/keycloak-at-kubecon-eu-2024-recap.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
+:title: Recap from KubeCon + CloudNativeCon Europe 2024
+:date: 2024-04-15
+:publish: false
```
Comment (stianst) [inline blog/2024/keycloak-at-kubecon-eu-2024-recap.adoc:3]:
```suggestion
:publish: true
```

**Review by stianst — APPROVED** [review]

**Pull request merged**